tests/Unit/Domain/DomainTestHelpers.hpp

```diff
@@ -38,8 +38,9 @@ class VolumeCornerIterator {
   void operator++() noexcept {
     ++index_;
     for (size_t i = 0; i < VolumeDim; i++) {
+      gsl::at(coords_of_corner_, i) = 2.0 * get_nth_bit(index_, i) - 1.0;
       gsl::at(array_sides_, i) =
-          2 * get_nth_bit(index_, i) - 1 == 1 ? Side::Upper : Side::Lower;
```

Comment:
leave this as it was (we don't like to compare doubles to ints)

Reply:
is there a test for VolumeCornerIterator?  If not, make one; if yes, test your new function